docs/topics/auth/default.txt

```diff
@@ -961,6 +961,13 @@ implementation details see :ref:`using-the-views`.
     * ``extra_context``: A dictionary of context data that will be added to the
       default context data passed to the template.
 
+    * ``redirect_authenticated_user``: Boolean. If True, authenticated users
+      accessing the login page will be redirected to
+      :setting:`settings.LOGIN_REDIRECT_URL <LOGIN_REDIRECT_URL>` or the ``next``
```

Comment:
You are right this should be sufficient as the interaction between LOGIN_REQUIRED and ``next`` is explained further down in the POST section